service/tracker/execution.go

```diff
@@ -61,7 +61,7 @@ func NewExecution(log zerolog.Logger, db *badger.DB, stream RecordStreamer) (*Ex
 		return nil, fmt.Errorf("could not retrieve root header: %w", err)
 	}
 	var sealID flow.Identifier
-	err = db.View(operation.LookupBlockSeal(blockID, &sealID))
+	err = db.View(operation.LookupBySealedBlockID(blockID, &sealID))
```

Comment:
`LookupBySealedBlockID` returns the seal for a specific block ID,

`LookupBlockSeal` was [renamed](https://github.com/onflow/flow-go/pull/2551/files#diff-bcdc07a0042c863f7276f619bf1e47447b89e28cff1809f725698633d1747b99L45) to `LookupLatestSealAtBlock` and returns the latest seal for a specific block ID
```suggestion
	err = db.View(operation.LookupLatestSealAtBlock(blockID, &sealID))
```

Reply:
Thanks!!